clang/include/clang/Basic/BuiltinsCheerp.def

```diff
@@ -28,4 +32,147 @@ BUILTIN(__builtin_cheerp_throw, "", "rT")
 BUILTIN(__builtin_cheerp_downcast, "", "T")
 BUILTIN(__builtin_cheerp_downcast_current, "", "T")
 
+// Query the current memory size, and increase the current memory size.
```

Comment:
here I see some unrelated builtins.
Remove the ones from here until my next comment (line 81)